{"name":"testPostCreateRepoOfUser","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"expected [200] but found [404]","trace":"java.lang.AssertionError: expected [200] but found [404]\n\tat org.testng.Assert.fail(Assert.java:94)\n\tat org.testng.Assert.failNotEquals(Assert.java:494)\n\tat org.testng.Assert.assertEquals(Assert.java:123)\n\tat org.testng.Assert.assertEquals(Assert.java:370)\n\tat org.testng.Assert.assertEquals(Assert.java:380)\n\tat actions.AssertActions.verifyStatusCode(AssertActions.java:9)\n\tat tests.PostCreateRepo.testPostCreateRepoOfUser(PostCreateRepo.java:29)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)\n\tat org.testng.internal.Invoker.invokeMethod(Invoker.java:714)\n\tat org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)\n\tat org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)\n\tat org.testng.TestRunner.privateRun(TestRunner.java:767)\n\tat org.testng.TestRunner.run(TestRunner.java:617)\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:334)\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:240)\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1149)\n\tat org.testng.TestNG.run(TestNG.java:1057)\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:110)\n"},"stage":"finished","description":"Verify that Status code is 200 when we Create a new GitHub Repo of User","steps":[],"attachments":[],"parameters":[],"start":1597242119948,"stop":1597242121460,"uuid":"0b9dbc05-18a9-4cab-ba0e-5defc85ad6f2","historyId":"59d2109dd25dba9af20471e98b7924bc","fullName":"tests.PostCreateRepo.testPostCreateRepoOfUser","labels":[{"name":"package","value":"tests.PostCreateRepo"},{"name":"testClass","value":"tests.PostCreateRepo"},{"name":"testMethod","value":"testPostCreateRepoOfUser"},{"name":"parentSuite","value":"Default Suite"},{"name":"suite","value":"TSAPIAutomation"},{"name":"subSuite","value":"tests.PostCreateRepo"},{"name":"host","value":"Pramods-MacBook-Pro.local"},{"name":"thread","value":"11732@Pramods-MacBook-Pro.local.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"},{"name":"owner","value":"Promode"},{"name":"severity","value":"normal"}],"links":[]}